src/components/homepage/Homepage.js

```diff
@@ -67,15 +68,24 @@ function Homepage({
         </li>
       </Header>
       <Hero onButtonClick={handleButtonClick} />
-      <Lead />
-      <HowToStart handleButtonClick={handleButtonClick} />
```

Comment:
I re-arranged three components (HowToStart, MessageContainer, and Requests) because they were out of order

Reply:
Oops.. that's hilarious, good catch 🥇